test/testutil/tests.c

```diff
@@ -416,3 +417,44 @@ int test_BN_abs_eq_word(const char *file, int line, const char *bns,
     BN_free(aa);
     return 0;
 }
+
+#define TIME_SIZE   18
+static void print_time(const ASN1_TIME *t, char *p)
+{
+    struct tm tm;
+
+    if (t == NULL) {
+        strncpy(p, "<null>", TIME_SIZE);
+    } else {
+        ASN1_TIME_to_tm(t, &tm);
+        sprintf(p, "%4d%02d%02d %02d:%02d:%02d", tm.tm_year + 1900,
```

Comment:
If you're *that* picky with str*n*cpy, then why not [BIO_]snprintf then? But is string value out of reach? I mean ASN1_TIME is a *printable* string...

Reply:
Opps, missed that.

Fix committed and will be squashed.